Desafio 02/desafio.js

```diff
@@ -0,0 +1,10 @@
+var val = ['o', undefined, null, 'l', undefined, 'a', ' null', null]
```

Comment:
Pode-se usar const nesse caso

Reply:
Alterado para const a declaração das variaveis que não vão mudar e dentro do for tinha um var para declarar a variavel 'n' foi alterada para let !!